js/picture.js

```diff
@@ -0,0 +1,205 @@
+  galleryOverlay.querySelector('.likes-count').textContent = caption[index].likes;
+  galleryOverlay.querySelector('.gallery-overlay-image').src = caption[index].url;
+  galleryOverlay.classList.remove('hidden');
+  document.addEventListener('keydown', onPictureEscPress);
+};
+
+var closePicture = function() {
+  galleryOverlay.classList.add('hidden');
+  document.removeEventListener('keydown', onPictureEscPress);
+};
+
+galleryClose.addEventListener('mousedown', function() {
+  closePicture();
+});
+
+galleryClose.addEventListener('keydown', function(evt) {
+  if (evt.keyCode === ENTER_KEYCODE) {
+    closePicture();
+  }
+});
+
+uploadCancel.addEventListener('click', function() {
+  uploadOverlay.classList.add('hidden');
+});
+
+document.addEventListener('keydown', function(evt) {
+  if (document.activeElement !== hashtagsForm && document.activeElement !== commentsForm && evt.keyCode === ESC_KEYCODE) {
+    uploadOverlay.classList.add('hidden');
+  }
+});
+
+var x = 1;
+resizeDecrease.addEventListener('mousedown', function() {
+  if (x > 1) {
+    x = 0.75;
+    effectImage.style.transform = 'scale('+ x +')';
+    uploadImageSize.value = (x * 100) + '%';
+  } else
+  x = x - 0.25;
+  if (x > 0) {
+    effectImage.style.transform = 'scale('+ x +')';
+    uploadImageSize.value = (x * 100) + '%';
+  }
+});
+
+resizeIncrease.addEventListener('mousedown', function() {
+  if (x < 0) {
+    x = 0.5;
+    effectImage.style.transform = 'scale('+ x +')';
+    uploadImageSize.value = (x * 100) + '%';
+  } else
+  x = x + 0.25;
+  if (x < 1.25) {
+    effectImage.style.transform = 'scale('+ x +')';
+    uploadImageSize.value = (x * 100) + '%';
+  }
+});
+
+effectNone.addEventListener('click', function() {
+  effectImage.style.filter = 'none';
+  effectLevel.classList.add('hidden');
+});
+
+effectChrome.addEventListener('click', function() {
+  effectImage.style.filter = 'grayscale(1)';
+  effectLevel.classList.remove('hidden');
+  effectLevelPin.style.transform = 'translateX(2000%)';
+});
+
+effectSepia.addEventListener('click', function() {
+  effectImage.style.filter = 'sepia(1)';
+  effectLevel.classList.remove('hidden');
+});
+
+effectMarvin.addEventListener('click', function() {
+  effectImage.style.filter = 'invert(100%)';
+  effectLevel.classList.remove('hidden');
+});
+
+effectPhobos.addEventListener('click', function() {
+  effectImage.style.filter = 'blur(3px)';
+  effectLevel.classList.remove('hidden');
+});
+
+effectHeat.addEventListener('click', function() {
+  effectImage.style.filter = 'brightness(3)';
+  effectLevel.classList.remove('hidden');
+});
+
+hashtagsForm.addEventListener('change', function() {
+  var valHash = hashtagsForm.value;
+  var arrHash = valHash.split(' ', 5);
+  var errorHash = false;
+  for (var hash of arrHash) {
+    if (hash[0] !== "#") {
+    hashtagsForm.setCustomValidity('хештег должен начинаться с #');
+    errorHash = true;
```

Comment:
```suggestion
      hashtagsForm.setCustomValidity('хештег должен начинаться с #');
      errorHash = true;
```
нужно выравнивание, скорее всего нужно проверить на ошибки линтера